ui/layouts/root-layout.tsx

```diff
@@ -1,17 +1,24 @@
 import React, { type ReactNode } from 'react';
 import cn from 'classnames';
 import { Header } from './header';
+import { removeGns } from './config';
 
 const width = 'max-w-[clamp(var(--width-sm),85vw,var(--width-max))]';
 const sidepanel = 'group-[.app--sidepanel]:max-w-[var(--width-max-sidepanel)]';
 
 export const RootLayout = ({ children }: { children: ReactNode }) => {
   return (
     <div className={cn('w-full h-full flex flex-col', width, sidepanel)}>
-      {/* Note: Consider a sticky header instead of overflow */}
-      <Header />
+      {removeGns ? (
```

Comment:
REMOVE_GNS is true, no need to handle conditional header at this level, it only matters for the home route